.gitmodules

```diff
@@ -102,3 +102,6 @@
 	path = helix-syntax/languages/tree-sitter-protobuf
 	url = https://github.com/yusdacra/tree-sitter-protobuf.git
 	shallow = true
+[submodule "helix-syntax/languages/tree-sitter-yaml"]
+	path = helix-syntax/languages/tree-sitter-yaml
+	url = https://github.com/ikatyang/tree-sitter-yaml
```

Comment:
Needs a `shallow = true`

Reply:
We should have a comment at the bottom saying need `shallow = true` or otherwise we need to comment on this every time.